cosmo_tester/test_suites/bootstrap_based_tests/ssl_test.py

```diff
@@ -21,11 +21,15 @@
 
 manager = bootstrap_based_manager
 DEFAULT_TENANT_ROLE = 'user'
+REMOTE_EXTERNAL_CERT_PATH = '/etc/cloudify/ssl/cloudify_external_cert.pem'
 
 
 def test_ssl(cfy, manager, module_tmpdir, attributes, ssh_key, logger):
     cert_path = join(module_tmpdir, '.cloudify', 'profiles',
```

Comment:
feels odd downloading it to the profile dir, maybe we should instead download it to just the tmpdir, since we're already calling `cfy profiles set -c` anyway?